code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -23,6 +23,29 @@
 	QDEL_NULL(modifier)
 	return ..()
 
+/datum/reagent/steroids
+	name = "Vaurca Pheromones"
+	id = "steroids"
+	description = "Pure, uncut, and unfiltered Vaurca Pheromones. The ultimate performance enhancer."
+	reagent_state = LIQUID
+	color = "#00BFFF"
+	overdose = 50
+	metabolism = 0.05 //A match is about 10 minutes. 30 units should last 10 minutes.
+	metabolism_min = 0.025
+	scannable = 1
+	taste_description = "testosterone"
+	var/datum/modifier/modifier01
+	var/datum/modifier/modifier02
```

Comment:
You need to `QDEL_NULL` both of these modifiers on `Destroy()`.

Reply:
Done.